src/main/java/org/datacleaner/kettle/configuration/DataCleanerConfigurationDialog.java

```diff
@@ -189,11 +186,7 @@ public void widgetSelected(SelectionEvent event) {
                 labelVersion.setText(VERSION.trim() + " " + editionDetails.getVersion());
             }
         } catch (Throwable e) {
-            String errorMessage = "There was an unexpected error launching DataCleaner";
-            if (e instanceof IOException) {
-                errorMessage = "The DataCleaner installation path could not be found.Please set the path in the menu Tools:DataCleaner configuration";
-            }
-            new ErrorDialog(Spoon.getInstance().getShell(), "Error launching DataCleaner", errorMessage, e);
+           //Do nothing 
```

Comment:
The exception could occur if the DataCleaner configuration  has never been set and therefore the file doesn't exist. I do not think the user needs to see it.